Dockerfile

```diff
@@ -14,7 +22,7 @@ RUN apt-get update && \
     apt-get -y remove libxml2-dev libudunits2-dev && rm -rf /var/lib/apt/lists/*
 
 # Volume and working directories
-VOLUME /glider-dac/logs/ /data /usr/local/lib/python3.11/site-packages/compliance_checker/data
+VOLUME /glider-dac/logs/ /data /usr/local/lib/python$PYTHON_VERSION_SHORT/site-packages/compliance_checker/data
```

Comment:
Possibly this cached Standard Name tables should live elsewhere within Compliance Checker, such as the directory specified by `XDG_CACHE_DIR`.